.editorconfig

```diff
@@ -65,6 +65,7 @@ dotnet_diagnostic.CA1003.severity = warning
 dotnet_diagnostic.CA1004.severity = warning
 dotnet_diagnostic.CA1005.severity = warning
 dotnet_diagnostic.CA1007.severity = warning
+dotnet_diagnostic.CA1008.severity = none
```

Comment:
REQ: Diese Regel nicht ausschalten, sondern den Wert `0` mit `Unknown` im entsprechenden `enum` ergänzen. Das wäre dann der Default und wir würden auf einen Blick sehen, wenn dieser irgendwo gesetzt werden würde, was wir nicht erwarten.

Reply:
Den Default haben wir indirekt doch schon mit der Nullable _srs_id_cli_ resp `OriginalReferenceSystem`?
Evtl. macht es Sinn die Regel direkt im Enum `ReferenceSystem` auszuschalten?

Reply:
Ich meine mit _Default_ nicht dasselbe wie du: Da `enum` ein Value Type ist, ist der Standardwert für ein nicht initialisiertes `ReferenceSystem`-Enum `0`. Daher sollte man, wenn möglich, den Wert `0` immer in einem `enum` definieren. Wenn der Wert `0` bzw. `Unknown/None` keinen Sinn im `enum` macht, dann sollte der meistverwendete Member (hier vermutlich `LV95`) den Wert `0` erhalten. Aktuell könnte beim Deserialisieren des Werts ein Fehler auftreten, wenn versucht wird `0` (z.B. Client liefert keinen gültigen Wert für das Enum ans API) zu einem `ReferenceSystem`-Enum zu deserialisieren.

Da wir momentan den Wert `0` nicht dem `LV95`-Member zuweisen können/wollen, müssen wir entweder `Unknown/None` einführen oder temporär die Regel im `enum` abstellen. Wenn wir die Regel abstellen, dann müssen wir kurz in einem Kommentar festhalten weshalb. Vielleicht können wir sie dann zu einem späteren Zeitpunkt wieder abstellen, wenn wir uns entscheiden, nicht mehr die Geolcode-IDs als Enum-Werte zu benutzen. Ich überlasse die Entscheidung dir.

Reply:
Geht für mich in Ordnung. 👍🏼 Ich gehe aber nicht davon aus, dass es beim deserialisieren zu irgendwelchen Problemen kommen könnte, da der Enum vom Typ _int_ per Definition irgendwelche int Values verträgt auch wenn diese nicht explizit definiert sind.